.github/workflows/ci.yml

```diff
@@ -108,8 +108,8 @@ jobs:
 
   size:
     if: |
-        ${{ github.event_name == 'pull_request'}} &&
-        ${{ github.event.pull_request.head.repo.full_name == 'home-assistant/iOS' }}
+      github.event_name == 'pull_request' &&
+      github.event.pull_request.head.repo.full_name == 'home-assistant/iOS'
```

Comment:
As per this discussion - https://github.com/orgs/community/discussions/25641#discussioncomment-3248571 - the key is to not use `{{` and `}}` 🤷